posts/2021-08-30-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc

```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title: "Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mezarin
+author_github: https://github.com/mezarin
+seo-title: Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster - OpenLiberty.io
+seo-description: Speed up development by making it easier for application developers to bind operator managed services to applications.
+blog_description: "Speed up development by making it easier for application developers to bind operator managed services to applications"
+open-graph-image: https://openliberty.io/img/blog/ol-stack-jpa-app-db-bind-browser-data-entry.png
+---
+= Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster
+Edward Mezarina <https://github.com/mezarin>
+
+Binding your Microservice application to operator based services (e.g. databases, etc.) has now been made easier through automatic collection of service data, deployment of needed resources, and data sharing with the application.
```

Comment:
suggested edit, something like:

Binding your microservice application to operator-based services , such as a database, is now easier, thanks to the Service Binding Operator. This operator enables  the automatic collection of service data, deployment of needed resources, and data sharing with the application.

Reply:
this introduction feels a little vague- I thin k it would be helpful to add a sentence or two to preview what the post will address, so the user knows what to expect. 

In your Gradle devfile post, you add this sentence:
"In this post, we’ll use the Open Liberty devfile stack and Openshift odo to deploy an application that uses Gradle to a Kubernetes cluster."
I think something along those lines, that mentions the goal and the primary tools would be helpful, and improve SEO for the post